src/Backend/Modules/Quotes/Installer/Installer.php

```diff
@@ -3,29 +3,30 @@
 namespace Backend\Modules\Quotes\Installer;
 
 use Backend\Core\Installer\ModuleInstaller;
+use Common\ModuleExtraType;
 
 final class Installer extends ModuleInstaller
 {
     public function install()
     {
         $this->addModule('Quotes');
 
-        $this->importSQL(dirname(__FILE__) . '/data/install.sql');
-        $this->importLocale(dirname(__FILE__) . '/data/locale.xml');
+        $this->importSQL(dirname(__FILE__) . '/Data/install.sql');
```

Comment:
the folder here is data so you need to change the casing or change the directory

Reply:
Fixed it in an amend